app/components/AddressBar/AddressBar.jsx

```diff
@@ -74,9 +75,8 @@ export default class AddressBar extends Component
 
         const addATab = ( tab ) =>
         {
-            addTab( { url: `safe-browser://${tab}`, isActiveTab: true } );
+            addTab( { url: `safe-browser://${tab}`, isActiveTab: true, windowId: remote.getCurrentWindow().webContents.id } );
```

Comment:
@manavbp , I thin we have windowId in the parent `Browser` component, so ideally we'd just pass that down and avoid having to import electron.remote here.